src/socialMediaLinkSharing.js

```diff
@@ -84,16 +125,76 @@ async function startMeasurement({
     reddit = false
 }) {
```

Comment:
Should replace the start and stop measurement conventions, since we don't use those anymore.